orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/rollingpush/CheckForRemainingTerminationsTask.groovy

```diff
@@ -33,6 +33,9 @@ class CheckForRemainingTerminationsTask implements Task {
       return TaskResult.SUCCEEDED
     }
 
-    return new TaskResult(ExecutionStatus.REDIRECT, [skipRemainingWait: false])
+    return new TaskResult(ExecutionStatus.REDIRECT, [
+      skipRemainingWait: false,
+      waitTaskState: [:]
```

Comment:
Not sure how this would have worked previously ... without reseting `waitTaskState` the wait task was just auto-completing after the first cycle.